sccp/imports/scRNA.py

```diff
@@ -14,71 +13,41 @@ def tensorFy(annD: anndata.AnnData, obsName: str) -> Pf2X:
     return Pf2X(data, sgUnique, annD.var_names)
 
 
-def import_perturb_RPE():
-    ds_disk = anndata.read_h5ad("/opt/andrew/rpe1_normalized_singlecell_01.h5ad")
-
-    # Remove NaNs
-    ds_disk = ds_disk[:, np.all(np.isfinite(ds_disk.X), axis=0)]
-
-    return tensorFy(ds_disk, "sgID_AB")
-
-
-def import_thompson_drug() -> anndata.AnnData:
-    """Imports cell readings from scRNA-seq data from PBMCs from PopAlign paper."""
-
+def ThompsonXA_SCGenes() -> Pf2X:
+    """Import Thompson lab PBMC dataset."""
     # Cell barcodes, sample id of treatment and sample number (33482, 3)
     metafile = pd.read_csv("sccp/data/Thomson/meta.csv")
 
     # Cell barcodes (33482)
-    barcodes = pd.read_csv(
-        "sccp/data/Thomson/barcodes.tsv", sep="\t", header=None, names=("cell_barcode",)
-    )
+    # barcodes = pd.read_csv(
+    #     "sccp/data/Thomson/barcodes.tsv", sep="\t", header=None, names=("cell_barcode",)
+    # )
 
     # Left merging should put the barcodes in order
-    metafile = pd.merge(
-        barcodes, metafile, on="cell_barcode", how="left", validate="one_to_one"
-    )
+    # metafile = pd.merge(
+    #     barcodes, metafile, on="cell_barcode", how="left", validate="one_to_one"
+    # )
 
     # h5ad is simplified version of mtx format
     # import scanpy as sc
     # data = sc.read_10x_mtx("./sccp/data/", var_names='gene_symbols', make_unique=True)
     # data.X = data.X.todense()
     # data = data[:, np.mean(data.X > 0, axis=0) > 0.001]
     # data.write('thompson.h5ad', compression="gzip")
-    data = anndata.read_h5ad("/opt/andrew/thomson.h5ad")
-
-    data.obs["Drugs"] = pd.Categorical(metafile["sample_id"])
-    return data
+    X = anndata.read_h5ad("/opt/andrew/thomson.h5ad")
 
-
-def ThompsonXA_SCGenes(offset: float = 1.0) -> anndata.AnnData:
-    """Import Thompson lab PBMC dataset."""
-    X = import_thompson_drug()
-    scalingfactor = 1000
+    X.obs["Drugs"] = pd.Categorical(metafile["sample_id"])
 
     assert np.all(np.isfinite(X.X.data))
 
-    X = X[:, np.mean(X.X > 0, axis=0) > 0.001]
     X.X /= np.sum(X.X, axis=0)
 
     # Only operating on the data works because 0 ends up as 0 here
-    X.X = np.log10((scalingfactor * X.X) + 1)
-    means = np.mean(X.X, axis=0)
-    cv = np.std(X.X, axis=0) / means
-
-    logmean = np.log10(means + 1e-10)
-    logstd = np.log10(cv + 1e-10)
-
-    if offset != 1.0:
```

Comment:
This is never used, and makes it look like this normalization is far more complex than it actually is.